waspc/data/Generator/templates/react-app/src/components/DefaultRootErrorBoundary.tsx

```diff
@@ -0,0 +1,13 @@
+import { useRouteError } from 'react-router-dom'
+
+import { FullPageWrapper } from './FullPageWrapper'
+
+export function DefaultRootErrorBoundary() {
```

Comment:
We are using a default root error boundary to make it look nicer when rendering fails. In the future, we should allow customisation of the root error boundary: https://github.com/wasp-lang/wasp/issues/2279